docs/user-manual/scripting/anatomy.md

```diff
@@ -3,7 +3,47 @@ title: Anatomy of a script
 sidebar_position: 3
 ---
 
-Here is a basic script. We can learn about the structure of a PlayCanvas script from it.
+Scripts provide powerful features for adding interactivity to your project. To learn more about the anatomy of a Script we've provided a simple `Rotate` script below. Let's break down what this means.
+
+:::note
+Scripts can be defined as either **[ES Modules](./esm-scripts.md)**, or **legacy scripts**.
+**[Learn more](./esm-scripts.md)**.
```

Comment:
Is there consensus, are they "legacy scripts" or "classic scripts"?

Also, second link seems wrong?